.github/workflows/automerge.yml

```diff
@@ -24,135 +22,27 @@ jobs:
-              env:
-                  GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
-
-            # This Slack step is duplicated in all workflows, if you make a change to this step, make sure to update all
-            # the other workflows with the same change
-            - uses: 8398a7/action-slack@v3
-              name: Job failed Slack notification
-              if: ${{ failure() }}
-              with:
-                  status: custom
-                  fields: workflow, repo
-                  custom_payload: |
-                      {
-                        channel: '#announce',
-                        attachments: [{
-                          color: "#DB4545",
-                          pretext: `<!here>`,
-                          text: `💥 ${process.env.AS_REPO} failed on ${process.env.AS_WORKFLOW} workflow 💥`,
-                        }]
-                      }
-              env:
-                  GITHUB_TOKEN: ${{ github.token }}
-                  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
-
-    staging:
-      runs-on: ubuntu-latest
-      needs: getPullRequestMergeability
-      if: github.event.pull_request.base.ref == 'staging' && github.actor == 'OSBotify' && github.event.label.name == 'automerge'
-      steps:
-        - name: Check for an auto approve
-          # Version: 2.0.0
-          uses: hmarr/auto-approve-action@7782c7e2bdf62b4d79bdcded8332808fd2f179cd
-          with:
-            github-token: ${{ secrets.GITHUB_TOKEN }}
-          if: github.event.pull_request.head.ref == 'main'
-
-        - name: Check for an auto merge
-          # Version: 0.12.0
-          uses: pascalgn/automerge-action@c9bd1823770819dc8fb8a5db2d11a3a95fbe9b07
-          env:
-            GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
-
-        # This Slack step is duplicated in all workflows, if you make a change to this step, make sure to update all
-        # the other workflows with the same change
-        - uses: 8398a7/action-slack@v3
-          name: Job failed Slack notification
-          if: ${{ failure() }}
-          with:
-            status: custom
-            fields: workflow, repo
-            custom_payload: |
-              {
-                channel: '#announce',
-                attachments: [{
-                  color: "#DB4545",
-                  pretext: `<!here>`,
-                  text: `💥 ${process.env.AS_REPO} failed on ${process.env.AS_WORKFLOW} workflow 💥`,
-                }]
-              }
-          env:
-            GITHUB_TOKEN: ${{ github.token }}
-            SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
-
-    production:
-      runs-on: ubuntu-latest
-      needs: getPullRequestMergeability
-      if: github.event.pull_request.base.ref == 'production' && github.actor == 'OSBotify' && github.event.label.name == 'automerge'
-      steps:
+        # TODO: Once https://github.com/hmarr/auto-approve-action/pull/186 is merged, point back at the non-forked repo
```

Comment:
Honestly, I'm not sure if just pointing at a forked repo like this will work, or if I need to checkout that repo in the workflow or publish the action in that repo or something

Reply:
Seems like this works just fine as-is: https://github.com/Andrew-Test-Org/Public-Test-Repo/pull/49/checks?check_run_id=2568980822